package.json

```diff
@@ -77,14 +77,14 @@
     "copy-to-clipboard": "^3.0.8",
     "crypto-js": "3.1.9-1",
     "electron-compile": "6.4.2",
-    "electron-devtools-installer": "2.1.0",
+    "electron-devtools-installer": "^2.2.4",
     "electron-json-storage": "4.0.2",
     "electron-squirrel-startup": "1.0.0",
     "file-saver": "1.3.8",
     "js-kinesis-sdk": "0.9.7",
     "react": "^16.4.1",
     "react-dom": "^16.4.1",
-    "react-hot-loader": "3.0.0-beta.6",
+    "react-hot-loader": "^4.3.4",
```

Comment:
are we actually using this? I keep getting errors in my console about how it hasn't been called from the root of the application, and when I add it there they stop. But it seems to be working fine with the reloading without putting it in there...